tasks/main.yml

```diff
@@ -3,6 +3,10 @@
 - name: Include user creation tasks
   ansible.builtin.include_tasks: user.yml
   when: slurm_create_user
+ 
+- name: Include Configure custom repositories
+  ansible.builtin.include_tasks: repositories-Debian.yml
+  when: slurm_configure_repos
```

Comment:
Should this support RHEL-like distros as well?
At least, this task should not execute when a host's family OS is not Debian.